test/FsAutoComplete.Tests.Lsp/TestCases/ExpectoTests/ExpectoTests.fsproj

```diff
@@ -1,8 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
testTheory is a feature of Expecto 10, which only supports .NET 6+, so I updated thise